### DISCUSSION TITLE
How do i change the name of the watermark?

### DISCUSSION THREAD

**Discussion by Noama12345** [danielkrupinski/Osiris#3098] [Q&A]
Please explane to me how to change the text in the watermark

**Comment by R3nzTheCodeGOD** [comment]
Find this line from Misc.cpp file and replace and compile

https://github.com/danielkrupinski/Osiris/blob/6e1bb02eeb7ce00a044f4c4002605c47dfa4254f/Osiris/Hacks/Misc.cpp#L404
